Stripe/StripeiOSTests/PaymentSheet+APITest.swift

```diff
@@ -287,7 +287,7 @@ class PaymentSheetAPITest: XCTestCase {
             }
 
             PaymentSheet.load(
-                clientSecret: IntentClientSecret.setupIntent(clientSecret: clientSecret),
+                retrievableIntent: RetrievableIntent.setupIntent(clientSecret: clientSecret),
```

Comment:
Can we add a test to load w/ a deferred Intent here?

Reply:
Added!